redis/redis_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2022 CloudWeGo Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package redis
+
+import (
+	"context"
+	"encoding/json"
+	"testing"
+
+	"github.com/cloudwego/hertz/pkg/app/server/registry"
+	"github.com/cloudwego/hertz/pkg/common/utils"
+	"github.com/go-redis/redis/v8"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	redisCli *redis.Client
+	ctx      = context.Background()
+)
+
+func init() {
+	rdb := redis.NewClient(&redis.Options{Addr: "127.0.0.1:6379"})
+	redisCli = rdb
+}
+
+// TestRegister Test the Registry in registry.go
+func TestRegister(t *testing.T) {
```

Comment:
The code is OK, but I find that the single test is not combined with Hertz to the extent that I'm not sure it's guaranteed to work with Hertz?

Reply:
I will enrich the tests combined with hertz, thanks for the reminder.

Reply:
done